src/OpenSage.Game.Tests/InstalledFilesTestData.cs

```diff
@@ -7,18 +7,18 @@ namespace OpenSage.Data.Tests
 {
     internal static class InstalledFilesTestData
     {
-        private static readonly IInstallationLocator locator;
+        private static readonly IInstallationLocator Locator;
 
         static InstalledFilesTestData()
         {
-            locator = new RegistryInstallationLocator();
+            Locator = new RegistryInstallationLocator();
         }
 
-        public static string GetInstallationDirectory(GameId game) => locator.FindInstallations(game).FirstOrDefault().Path;
+        public static string GetInstallationDirectory(SageGame game) => Locator.FindInstallations(game).FirstOrDefault().Path;
```

Comment:
I think this should be `FirstOrDefault()?.Path`.

Reply:
I can change it, although it won't make a _huge_ difference - the test will crash anyway if the game files can't be found, but the stack trace will point to the usage site of the return value instead of here.

Reply:
In that case, could you change it to `First().Path`, then it will still crash, but with a slightly more meaningful exception :)